AngularLanguageService.Shared/LanguageServer/AngularConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace AngularLanguageService.Shared.LanguageServer
+{
+    internal static class AngularConstants
+    {
+        internal const string TypeScriptContentTypeName = "TypeScript";
+
+        internal const string AngularComponentContentTypeName = "angularcomponent";
+        internal const string AngularComponentDelegationContentTypeName = "angularcomponent-delegation";
```

Comment:
nit: the -delegation postfix is a detail of the HTML LSP implementation in VS.  It's not really relevant to the Angular implementation.  If the content types can be consolidated (see other comment), this wouldn't be necessary.

Reply:
I was just using the `-delegation` postfix to be clear about the different between the content types (which relies on one of them inheriting from `html-delegation`) and so I was just following that "convention". I'll follow your recommendation of using `#if` directives though.